src/System.AppContext/src/System.AppContext.csproj

```diff
@@ -22,7 +22,8 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net463_Release|AnyCPU'" />
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)'=='true'">
-    <TargetingPackReference Include="mscorlib" />
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' == 'net463'" />
```

Comment:
Hmmm, the TargetingPackReference for mscorlib is set in like hundreds more files than what I am touching in this change. I will do it as a follow sweep PR to remove it from everywhere.